packages/contracts/src/constants.ts

```diff
@@ -45,39 +46,84 @@ export const foxEthStakingContractAddresses = [
+export const TS_AGGREGATOR_TOKEN_TRANSFER_PROXY_MAINNET =
+  '0xF892Fef9dA200d9E84c9b0647ecFF0F34633aBe8'
 
 // RFOX on Arbitrum ERC1967Proxy contract address
 // Uncomment me if you want to test RFOX with a shorter cooldown period
 // export const RFOX_PROXY_CONTRACT_ADDRESS: Address = '0x1094c4a99fce60e69ffe75849309408f1262d304'
 export const RFOX_PROXY_CONTRACT_ADDRESS: Address = '0xac2a4fd70bcd8bab0662960455c363735f0e2b56'
-
 export const RFOX_REWARD_RATE = 1n * 10n ** 27n
 export const RFOX_WAD = 1n * 10n ** 18n
 
 export const CONTRACT_ADDRESS_TO_ABI = {
-  [ETH_FOX_POOL_CONTRACT_ADDRESS]: IUniswapV2Pair,
-  [ETH_FOX_STAKING_CONTRACT_ADDRESS_V1]: FarmingABI,
-  [ETH_FOX_STAKING_CONTRACT_ADDRESS_V2]: FarmingABI,
-  [ETH_FOX_STAKING_CONTRACT_ADDRESS_V3]: FarmingABI,
-  [ETH_FOX_STAKING_CONTRACT_ADDRESS_V4]: FarmingABI,
-  [ETH_FOX_STAKING_CONTRACT_ADDRESS_V5]: FarmingABI,
-  [ETH_FOX_STAKING_CONTRACT_ADDRESS_V6]: FarmingABI,
-  [ETH_FOX_STAKING_CONTRACT_ADDRESS_V7]: FarmingABI,
-  [ETH_FOX_STAKING_CONTRACT_ADDRESS_V8]: FarmingABI,
-  [ETH_FOX_STAKING_CONTRACT_ADDRESS_V9]: FarmingABI,
-  [FOX_TOKEN_CONTRACT_ADDRESS]: erc20ABI,
-  [UNISWAP_V2_ROUTER_02_CONTRACT_ADDRESS]: IUniswapV2Router02,
+  [ETH_FOX_POOL_CONTRACT_ADDRESS]: UNISWAP_V2_PAIR_ABI,
+  [ETH_FOX_STAKING_CONTRACT_ADDRESS_V1]: FARMING_ABI,
+  [ETH_FOX_STAKING_CONTRACT_ADDRESS_V2]: FARMING_ABI,
+  [ETH_FOX_STAKING_CONTRACT_ADDRESS_V3]: FARMING_ABI,
+  [ETH_FOX_STAKING_CONTRACT_ADDRESS_V4]: FARMING_ABI,
+  [ETH_FOX_STAKING_CONTRACT_ADDRESS_V5]: FARMING_ABI,
+  [ETH_FOX_STAKING_CONTRACT_ADDRESS_V6]: FARMING_ABI,
+  [ETH_FOX_STAKING_CONTRACT_ADDRESS_V7]: FARMING_ABI,
+  [ETH_FOX_STAKING_CONTRACT_ADDRESS_V8]: FARMING_ABI,
+  [ETH_FOX_STAKING_CONTRACT_ADDRESS_V9]: FARMING_ABI,
+  [FOX_TOKEN_CONTRACT_ADDRESS]: ERC20_ABI,
+  [UNISWAP_V2_ROUTER_02_MAINNET]: UNISWAP_V2_ROUTER_02_ABI,
   // THOR Router Mainnet
-  [THOR_ROUTER_CONTRACT_ADDRESS_ETHEREUM]: THORChain_RouterABI,
+  [THOR_ROUTER_CONTRACT_ADDRESS_MAINNET]: THORCHAIN_ROUTER_ABI,
 } as const
 
 export const CONTRACT_TYPE_TO_ABI = {
-  [ContractType.UniV2Pair]: IUniswapV2Pair,
-  [ContractType.ERC20]: erc20ABI,
-  [ContractType.ThorRouter]: THORChain_RouterABI,
+  [ContractType.UniV2Pair]: UNISWAP_V2_PAIR_ABI,
+  [ContractType.ERC20]: ERC20_ABI,
+  [ContractType.ThorRouter]: THORCHAIN_ROUTER_ABI,
 } as const
+
+// 0x proxy contracts
+export const ZRX_ARBITRUM_PROXY_CONTRACT = '0xDef1C0ded9bec7F1a1670819833240f027b25EfF'
+export const ZRX_AVALANCHE_PROXY_CONTRACT = '0xDef1C0ded9bec7F1a1670819833240f027b25EfF'
+export const ZRX_BASE_PROXY_CONTRACT = '0xDef1C0ded9bec7F1a1670819833240f027b25EfF'
+export const ZRX_BSC_PROXY_CONTRACT = '0xDef1C0ded9bec7F1a1670819833240f027b25EfF'
+export const ZRX_ETHEREUM_PROXY_CONTRACT = '0xDef1C0ded9bec7F1a1670819833240f027b25EfF'
+export const ZRX_OPTIMISM_PROXY_CONTRACT = '0xDEF1ABE32c034e558Cdd535791643C58a13aCC10'
```

Comment:
Could potentially move this one last, and assign all others to `ZRX_ETHEREUM_PROXY_CONTRACT` with a comment explaining that all except for OP leverage the same contract addy

Reply:
heyy free cleanup whilst in da house 🎉 https://github.com/shapeshift/web/pull/7687/commits/864cf26c60ff7e887e54e9ee589c657ae8381222